http-server/src/main/java/org/triplea/server/http/AppConfig.java

```diff
@@ -0,0 +1,32 @@
+package org.triplea.server.http;
```

Comment:
This is an important class to be aware of. DropWizard serializes the YML configuration file values to this class and then injects this class as a bean to the main application. The YML configuration file is specified as the second argument to the main method that launches drop wizard, see the application class for more details: https://github.com/triplea-game/triplea/pull/4846/files#diff-69b1b6a2651245b3f3d83460de5856fd